bin/backstop

```diff
@@ -20,19 +20,28 @@ ROOT_DIR=$(dirname "$BIN_DIR")
 VERSION=6.1.1
 IMAGE=googlesitekit/vrt:$VERSION
 
-# Build the backstopjs image if it doesn't exist yet.
-docker image inspect $IMAGE -f "{{.Id}}" >/dev/null 2>&1 ||
-	docker build -t $IMAGE --build-arg BACKSTOPJS_VERSION=$VERSION "$ROOT_DIR/docker/backstopjs"
+# (re)Build the backstopjs image as needed.
+docker build \
+	--pull \
```

Comment:
This fixes the problem of people getting different results due to changes in some of the source images used. It's not perfect since we don't install fixed versions of packages with apk but okay for now.